pagopa/io-dev-api-server
# feat: [IOBP-652,IOBP-656] Add biz-events transaction download pdf

## ⚠️ This PR depends on #369

## Short description
This PR adds a new endpoint to download a biz-events transaction pdf

## List of changes proposed in this pull request
- Added a new endpoint `GET /transactions/:transactionId/pdf` that returns a lorem ipsum pdf document

## How to test
With postman try to execute the API request above of a transaction, you should be able to download a pdf receipt
